run/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.decorators import api_view
+from rest_framework.parsers import JSONParser
+from .serializers import RunSerializer
+from .models import Run
+from rest_framework.response import Response
+
+
+@api_view(http_method_names=['POST'])
```

Comment:
To use the api, a user submit a list of runs. For each run we validate it and if it's valid we have to give the user an identifier so that it can later reference the created run with it and for example query the results. Without it the user won't be able to map the completed runs in our system to the request in their system.

Also, you are returning the whole run data if it wasn't validated. Simply return an error with a description. Returning the data provided to us by the user isn't useful to anyone.